browse-scaler/src/resize.js

```diff
@@ -1,5 +1,5 @@
-const sharp = require('sharp');
-const fetch = require('isomorphic-unfetch');
+const sharp = require("sharp");
+const fetch = require("isomorphic-unfetch");
```

Comment:
This was removed from the dependencies in `package.json`. Is it an implicit dependency of `node-fetch`?